src/server/rest/service/TenantService.ts

```diff
@@ -75,6 +75,20 @@ export default class TenantService {
     next();
   }
 
+  public static async handleGetTenantLogo(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    // Filter
+    const tenantID = TenantSecurity.filterTenantRequestByID(req.query);
+    UtilsService.assertIdIsProvided(action, tenantID, MODULE_NAME, 'handleGetTenantLogo', req.user);
+    // Get it
+    const tenantLogo = await TenantStorage.getTenantLogo(tenantID);
+    // Check
+    UtilsService.assertObjectExists(action, tenantLogo, `Tenant with ID '${tenantID}' does not exist`,
```

Comment:
Do like in UserService:
- Retrieve the Tenant by ID first and check if the tenant exists.
- Next, get the logo from DB and return it without checking.
Do the same in CompanyService, SiteService and SiteAreaService.